draft-ietf-quic-http.md

```diff
@@ -1868,7 +1861,9 @@ use the full 32-bit space.  Settings ported from HTTP/2 might choose to redefine
 the format of their settings to avoid using the 62-bit encoding.
 
 Settings need to be defined separately for HTTP/2 and HTTP/3. The IDs of
-settings defined in {{!RFC7540}} have been reserved for simplicity. See
+settings defined in {{!RFC7540}} have been reserved for simplicity.  Note that
+the settings identifier space in HTTP/3 is substantially larger (62 bits versus
+16 bits), so many HTTP/3 settings have no equivalent HTTP/2 code point. See
```

Comment:
Unfortunately, `0x1` is reserved by QPACK so it makes a little more difficult to see that this is true. And a quick check of the [HTTP/2 IANA registry](https://www.iana.org/assignments/http2-parameters/http2-parameters.xhtml#settings) shows that 0x0 is also reserved, but I can't seem to find where that is defined in RFC 7540.

Checking that, it seems we have a clash on 0x8 due to RFC 8441 - WebSocket over HTTP/2. I think its reasonable that someone might write a WebSocket over HTTP/3 extension and reusing the code point would be nice. Can we move NUM_PLACEHOLDERS to 0x9 and leave 0x8 open?

Reply:
#2443 (that feature that creates an issue is nice, but it should leave a tombstone so that people know not to reply).

Reply:
Seconded.  GitHub feedback submitted.